index.html

```diff
@@ -7,37 +7,194 @@
+                </ul>
 
+                <button class="btn">See Project</button>
+            </div>
+        </article>
+        <article class="Card">
+            <img src="./img/Snapshoot3.svg" alt="">
+            <div class="TitleContainer">
+                <div class="Title">
+                    <h4>Tonic</h4>
+                    <div class="Campoy">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                </div>
+                <p class="Info">
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+                <ul class="CusList">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+
+                <button class="btn">See Project</button>
+            </div>
+        </article>
+        <article class="Card h">
+            <img src="./img/Snapshoot4.svg" alt="">
+            <div class="TitleContainer">
+                <div class="Title">
+                    <h4>Multi-Post
+                        Stories</h4>
+                    <div class="Campoy">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                </div>
+                <p class="Info">
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+                <ul class="CusList">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+
+                <button class="btn">See Project</button>
+            </div>
+        </article>
+    </section>
+
+    <section class="About-Myself">
+        <article class="AboutContainer">
+            <div class="AboutTop">
+                <article>
+                    <h2>About Myself</h2>
+                    <p>Hello I’m a software developer! I can help you build a product,
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.</p>
+                </article>
+                <h4>Lets connect</h4>
+                <ul>
+                    <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-github" aria-hidden="true"></i></li>
```

Comment:
Please ensure that each section has the margins (top, bottom, left, right) defined in the design, and the font and colors (background and font) defined for the template according to the project requirement.